source/jsonnet/england-wales/household/blocks/who-lives-here/anyone_else.jsonnet

```diff
@@ -0,0 +1,177 @@
+local rules = import '../../../lib/common_rules.libsonnet';
+local placeholders = import '../../lib/placeholders.libsonnet';
+
+local questionTitle(isPrimary) = (
+  if isPrimary then {
+    text: 'Do any of the following people live at {household_address} on {census_date}?',
+    placeholders: [
+      placeholders.address,
+      placeholders.censusDate,
+    ],
+  } else {
+    text: 'Do any of the following people also live at {household_address} on {census_date}?',
+    placeholders: [
+      placeholders.address,
+      placeholders.censusDate,
+    ],
+  }
+);
+
+local anyoneElseAnswerExclusiveText(isPrimary) = (
+  if isPrimary then {
+    label: 'None of these apply, I am the only person who usually lives here',
+    value: 'None of these apply, I am the only person who usually lives here',
+  } else {
+    label: 'None of these apply, no-one usually lives here',
+    value: 'None of these apply, no-one usually lives here',
+    description: 'For example, this is a second address or holiday home',
+  }
+);
+
+local question(isPrimary) = {
+  type: 'MutuallyExclusive',
+  id: 'anyone-else-driving-question',
```

Comment:
Don't think we need the driving in the id, it doesn't mean anything downstream and how it was collected is arbitrary

Reply:
there is already an `anyone-else` block, so dropping the driving would result in an invalid schema (assuming we keep the convention of appending block ids with `-question`).

Open to other suggestions though

Reply:
That actually makes it worse, we shouldn't be distinguishing between blocks by the way its collected. The questions is aimed at capturing people who are often here, as the question says 
```This is often their permanent or family home.``

With that in mind I would take the key words and go with something like often-permanent-home